README.rst

```diff
@@ -29,15 +29,33 @@ Usage:
 
     jinja2_standalone_compiler <path-to-jinja-files>  # Note that jinja files will be searched recursively
 
-For a more intense usage, you can also use a settings file to define a bunch of things. To use settings file:
+* For a more intense usage, you can also use a settings file to define a bunch of things. To use settings file:
 
-.. code:: bash
+    .. code:: bash
+
+        jinja2_standalone_compiler <path-to-jinja-files> --settings=path/to/settings.py
+
+    Please note that settings file is a Python file, so Python syntax applies. =P
```

Comment:
Doesn't look like code, but got an indent (which was intended). Couldn't find a nice way of doing that... ideas welcome :)